Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestStepViewController.swift

```diff
@@ -71,6 +86,93 @@ class TestStepViewController: UIViewController, UITextViewDelegate {
         navigationController?.popViewController(animated: true)
     }
     
+    
+    @IBAction func didTapActionHeader(_ sender: Any) {
+        expandActionTextView()
+    }
+    
+    @IBAction func didTapExpectedResultsHeader(_ sender: Any) {
+        expandExpectedResultsTextView()
+    }
+    
+    
+    @IBAction func didTapNotesHeader(_ sender: Any) {
+        expandNotesTextView()
+    }
+    
+    func alignHeaderButtonContents() {
+        let imageInset = UIEdgeInsets(top: 0, left: 10, bottom: 0, right: 0)
+        let titleInset = UIEdgeInsets(top: 0, left: 10, bottom: 0, right: 0)
+        //Set the insets for all the accordion buttons to layout the image and title of the button
+        actionButton.imageEdgeInsets = imageInset
+        actionButton.titleEdgeInsets = titleInset
+        expectedResultButton.imageEdgeInsets = imageInset
+        expectedResultButton.titleEdgeInsets = titleInset
+        notesButton.imageEdgeInsets = imageInset
+        notesButton.titleEdgeInsets = titleInset
+    }
+    
+    func expandActionTextView() {
+        //If the text view is showing collapse, otherwise expand in remaining space and set the button image to down arrow
+        if actionTextViewHeightContraint.constant != 0 {
+            actionTextViewHeightContraint.constant = 0
+            actionButton.setImage(UIImage.init(named: rightArrowStr), for: .normal)
+        } else {
+            //Make the text view as large as it can be while still fitting in its superview.
+            actionTextViewHeightContraint.constant = (actionTextView.superview?.frame.height)! * textViewHeightRatio - titleAndDividerHeight - totalSpaceBetweenButtons
+            actionButton.setImage(UIImage.init(named: downArrowStr), for: .normal)
+        }
+        //Set the other text views heights to 0
+        expectedResultsTextViewHeightConstraint.constant = 0
+        notesTextViewHeightConstraint.constant = 0
+        //Set the other buttons images to right chevrons
+        expectedResultButton.setImage(UIImage.init(named: rightArrowStr), for: .normal)
+        notesButton.setImage(UIImage.init(named: rightArrowStr), for: .normal)
+        //scroll the text view to the top
+        actionTextView.scrollRectToVisible(CGRect.zero, animated: false)
+    }
+    
+    func expandExpectedResultsTextView() {
+        //If the text view is showing collapse, otherwise expand in remaining space and set the button image to down arrow
+        if expectedResultsTextViewHeightConstraint.constant != 0 {
```

Comment:
It looks like this if/else section is reused repeatedly.  Is it possible to extract it as a method that takes the target textview as a parameter?